### PR TITLE
Update example.jl

### PR DESCRIPTION
**Pull request by greimel**
<s> ... see my suggestion in #35 </s> I hadn't submitted my review before the PR got merged.

### CODE REVIEW

**Comment by ayushpatnaikgit** [comment]
@chiraganand should `asset_path` function be inside `load_data`? 

`load_data` is the only function that will use `asset_path`

**Comment by ayushpatnaikgit** [comment]
> ... see my suggestion in #35 I hadn't submitted my review before the PR got merged.

Apologies, @greimel

**Comment by greimel** [comment]
> Apologies, @greimel 

no worries - it was my bad

**Comment by greimel** [comment]
> @chiraganand should `asset_path` function be inside `load_data`? 
> 
> 
> 
> `load_data` is the only function that will use `asset_path`

this wouldn't be idiomatic in Julia

**Comment by ayushpatnaikgit** [comment]
> 

We've been considering doing this for various functions. For example, functions `mean` and `SE` inside `svymean` and `sum` and `SE` (different) inside `svytotal`. We haven't been able to decide, and I was trying to use this PR as an opportunity for that. 

 Do you have any references on this?

**Comment by greimel** [comment]
I've contributed to multiple packages and I've never seen this pattern – unless you create closures.

I guess the issue is that the inner function would have to be recompiled at every function call. If the function is outside, then it is compiled only once.

**Comment by ayushpatnaikgit** [comment]
> I've contributed to multiple packages and I've never seen this pattern – unless you create closures.
> 
> I guess the issue is that the inner function would have to be recompiled at every function call. If the function is outside, then it is compiled only once.

@ViralBShah can you please guide us on this? We are discussing whether we should nest functions or not.

**Comment by ViralBShah** [comment]
Unless it is really crucial to use inner functions, I would avoid them.

**Comment by KristofferC** [comment]
> I guess the issue is that the inner function would have to be recompiled at every function call. If the function is outside, then it is compiled only once.

This is not true. It gets compiled once in any case.

**Comment by chiraganand** [comment]
> @chiraganand should `asset_path` function be inside `load_data`?
> 
> `load_data` is the only function that will use `asset_path`

It shouldn't matter even if only one function is using it one can still keep the function outside. Given that `load_data` just loads data I am guessing there are no performance issues to deal with in which case `@inline` could be explored.

**Review by ayushpatnaikgit — APPROVED** [review]
Regardless of the nesting debate, this PR is correct and good to merge.

File: src/example.jl [hunk 1]
```diff
@@ -20,8 +20,7 @@ The API program has been discontinued at the end of 2018. Information is archive
 """
 function load_data(name)
     name = name * ".csv"
-    assets_path = joinpath(PKG_DIR, "assets")
```
Comment (iuliadmtru) [inline src/example.jl:23]: This can be removed. I did not see the function on line 2 when I added this.

**Pull request merged**